### PR TITLE
Add example of Nyikos of first countable, T1 , weakly Lindelof, sequentially compact space which is not also compact

### PR DESCRIPTION
**Pull request by StevenClontz**
https://math.stackexchange.com/questions/4874239/is-there-a-first-countable-t-1-weakly-lindelof-sequentially-compact-space-w/4874321#4874321

### CODE REVIEW

**Comment by prabau** [comment]
Regarding the title, the underlying set is $\omega_1$, so there is no long ray involved here.

The topology is the join of the cofinite topology and the left ray topology on $\omega_1$ in the lattice of all topologies on that set.  So instead of "Modified ...", the name "Join of cofinite and left ray topologies on $\omega_1$" would be a good one to precisely identify the space.

In the README file, it's not clear a reader would fully understand.  Need to specify what $\alpha$ is supposed to be.
Also, can we add the mathse post as a reference for the README?

P20: the justification given here is not quite right.

Can we add the trait P114 (cardinality $\aleph_1$)?

**Comment by StevenClontz** [comment]
Think I've addressed all except

> P20: the justification given here is not quite right.

I gues I don't see the error: we're in a well-founded LOTS, so take the subsequence of strictly increasing points; if we're non-trivial this is infinite and converges to the supremum, I believe.

**Comment by prabau** [comment]
For P20 what do you mean by a trivial sequence?  I guess you mean a sequence that is eventually constant.  But if that is what is meant, then it is not true that there is a sequence that converges to the supremum of its range.  For example, the sequence $(3,1,2,1,2,1,2,...)$ is not trivial in the sense above, has a range with supremum $3$ and has no subsequence converging to $3$.

Now I know what you mean to say, but even for non-trivial sequences there may not be a subsequence of strictly increasing points. Writing a full non-sloppy justification would be something like, "if there is a value that appears infinitely many often, the corresponding constant subsequence converges.  Otherwise, replacing the sequence by a subsequence, we can assume the elements of the sequence are all distinct.  By the well-orderedness of $\omega_1$ the range of the sequence has a minimum value $x_{n_0}$ at index $n_0$.  Then take index $n_1>n_0$ with $x_{n_1}=\min\\{x_n:n>n_0\\}$. etc etc.  Continuing in this way, we get a strictly increasing subsequence $(x_{n_k})$ of $(x_n)$, etc etc"

That explanation is way too long winded though.  So why don't we just repeat the correct justification from Ulli, with the easy details swept under the "wlog":

> Let $(x_n)_{n < \omega}$ be a sequence in $\omega_1$.  Wlog we may assume $(x_n)_n$ has pairwise distinct values and is strictly increasing.  Then it converges to its supremum, which belongs to $\omega_1$.

**Comment by StevenClontz** [comment]
Thanks for catching the oversight. I'll address it one way or another, probably later today.

File: spaces/S000195/properties/P000020.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000195
+property: P000020
+value: true
+refs:
+    - mathse: 4874321
+      name: Answer to "Is there a first countable, $T_1$, weakly Lindelof, sequentially compact space which is not also compact?"
+---
+
+Every sequence has a monotonic increasing subsequence that converges to the supremum of its points.
```
Comment (StevenClontz) [inline spaces/S000195/properties/P000020.md:10]: This handles sequences with constant subsequences, as well as those without.

**Comment by prabau** [comment]
Hmm, does "monotone increasing" mean strictly increasing, or just non-decreasing?

**Review by prabau — APPROVED** [review]

**Pull request merged**